github_client/github.go

```diff
@@ -1,11 +1,12 @@
-package github
+package github_client
```

Comment:
Bug: The dependency "github.com/ProtonMail/go-crypto" is being pointed at a development version. Please ensure a stable version is used as the changes introduced there may break this code in future.